src/main/java/io/mathdojo/model/Question.java

```diff
@@ -0,0 +1,70 @@
+package io.mathdojo.model;
+
+import org.springframework.data.annotation.Id;
+
+import com.microsoft.azure.spring.data.cosmosdb.core.mapping.Document;
+
+@Document(collection = "questions")
+public class Question {
+	
+	@Id
+    private String questionTitle;
+    private String questionBody;
+    private String sampleAnswer;
+    private String[] hints;
+
+    public Question() {
+    }
+
+    public Question(String questionTitle, String questionBody, String sampleAnswer, String[] hints) {
+        this.questionTitle = questionTitle;
+        this.questionBody = questionBody;
+        this.sampleAnswer = sampleAnswer;
+        this.hints = hints;
+    }
+
+	public Question(Question question) {

```

Comment:
Why do we have a Question constructor that takes in Question objects?

Reply:
I think I was testing something. I've removed that